### PR TITLE
[Notifier] Updated chatter documentation with `Telegram::answerCallbackQuery`

### LINKED ISSUE #17501: [Notifier] [Telegram] Add support to answer callback queries
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#48374
| PR author(s) | @alexsoft
| Merged in    | 6.3

### PR DESCRIPTION
**Pull request by alexsoft**
Adding documentation for the new feature of Telegram notifier: answer callback query.
Fixes #17501

### CODE REVIEW

File: notifier/chatters.rst [hunk 1]
```diff
@@ -332,6 +332,28 @@ a previous message to edit::
             ])
         );
 
+Answering Callback Queries in Telegram
+~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (OskarStark) [inline notifier/chatters.rst]:
```suggestion
Answering Callback Queries in Telegram
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: notifier/chatters.rst [hunk 2]
```diff
@@ -332,6 +332,28 @@ a previous message to edit::
...
+    use Symfony\Component\Notifier\Bridge\Telegram\TelegramOptions;
+    use Symfony\Component\Notifier\Message\ChatMessage;
+
+    $chatMessage = new ChatMessage('Thank you!');
+    $telegramOptions = (new TelegramOptions())
+        ->chatId($chatId)
+        ->answerCallbackQuery(
+            $callbackQueryId = '12345', // extracted from callback
+            $showAlert = true,
+            $cacheTime = 1,
```
Comment (OskarStark) [inline notifier/chatters.rst]:
```suggestion
            callbackQueryId: '12345', // extracted from callback
            showAlert: true,
            cacheTime: 1,
```
WDYT of using named arguments here?

**Review by OskarStark — COMMENTED** [review]
Some comments, afterwards good to merge 👍

File: notifier/chatters.rst (hunk 1 above)
Comment (alexsoft) [reply notifier/chatters.rst]: I don't quite understand why amount of `~` characters here should be more than in _Updating Telegram Messages_ title case above

File: notifier/chatters.rst (hunk 2 above)
Comment (alexsoft) [reply notifier/chatters.rst]: Yes, I like it, thank you

File: notifier/chatters.rst (hunk 1 above)
Comment (OskarStark) [reply notifier/chatters.rst]: For RST it just needs a minimum of three characters (IIRC), but in the Symfony docs we make as long as the title itself

**Review by OskarStark — APPROVED** [review]

File: notifier/chatters.rst (hunk 1 above)
Comment (alexsoft) [reply notifier/chatters.rst]: Oh, ok. Somehow I did not get it. Thank you, updated the file

**Comment by OskarStark** [comment]
Thanks Alex for working on this feature, this is much appreciated.

**Comment by alexsoft** [comment]
Thank you very much, Oskar, for reviewing and helping with PRs.

**Pull request merged**